packages/Input/Input.md

```diff
@@ -14,12 +14,19 @@ For information on how to use disabled Inputs, please refer to the [disabled for
 ```jsx
 <Box between={2}>
   <Input label="First name" value="Harry" />
-
   <Input type="tel" label="Mobile phone" pattern="[0-9]{10}" maxLength="10" />
   <Input type="password" label="Tax ID" value="123456789" autoComplete="on" />
 </Box>
 ```
 
+### Usage criteria
+
+- Use `hintPosition` to re-position the `hint` below the `label` if the inline `hint` wraps
+  - The `hint` should **not** be longer than 85 characters
+  - The `hint` should **not** wrap more than 3 lines on all viewport sizes
+- We recommend `React.Fragment` when supplying custom strings to the `error` prop
+  - Note that the `div` tag is not valid in a `p` tag
```

Comment:
Let's add some more notes here:

- Keep `error` text as brief as possible, should be limited to text and links